sdk/ai/ai-projects/src/agents/index.ts

```diff
@@ -138,26 +137,25 @@ export interface AgentsOperations {
 
   /** Gets a list of previously uploaded files. */
   listFiles: (
-    options?: ListFilesParameters,
+    purpose?: FilePurpose, requestParams?: OptionalRequestParameters
   ) => Promise<FileListResponseOutput>;
   /** Uploads a file for use by other operations. */
-  uploadFile: (
-    options: UploadFileParameters,
-  ) => Promise<OpenAIFileOutput>;
+  uploadFile: (content: ReadableStream |NodeJS.ReadableStream, purpose:FilePurpose, fileName?: string, requestParams?: OptionalRequestParameters   
```

Comment:
Should content parameter be named 'content' or 'file' (RE [the typespec](https://github.com/Azure/azure-rest-api-specs/blob/0d2ca01a170641d8b1f5dace38b8995bb7563be8/specification/ai/Azure.AI.Projects/agents/files/routes.tsp#L58))

Reply:
I think this was changed so the stream input could be read from the file in the sample, but I was having some other issues there so we ended up removing it. I see your point in matching the typespec. @ganeshyb should this be changed to file or left as content?

Reply:
file would be restricted name, CSharp had it as data, so we can change that as well.